{"name": "test_update_cart_quantity_3", "status": "failed", "statusDetails": {"message": "pytest_assume.plugin.FailedAssumption: \n1 Failed Assumptions:\n\ncase\\test_update_cart_quantity.py:94: AssumptionFailure\n>>\tpytest.assume(GetKeyword.get_keyword(result, \"message\") != \"操作成功\", \"断言开发自检字段\")\nAssertionError: 断言开发自检字段\nassert False", "trace": "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               1 Failed Assumptions:\nE               \nE               case\\test_update_cart_quantity.py:94: AssumptionFailure\nE               >>\tpytest.assume(GetKeyword.get_keyword(result, \"message\") != \"操作成功\", \"断言开发自检字段\")\nE               AssertionError: 断言开发自检字段\nE               assert False\n\nd:\\python\\lib\\site-packages\\six.py:702: FailedAssumption"}, "description": "\n    修改商品数量超过库存数量\n    :return:\n    ", "start": 1615079568568, "stop": 1615079568567, "uuid": "b8495df2-c3ef-48e5-87bf-fac6ca771b69", "historyId": "699307ffe872d29b570cd08c09c4357a", "testCaseId": "e6e49bf24c55442cc3f9d98896370628", "fullName": "case.test_update_cart_quantity#test_update_cart_quantity_3", "labels": [{"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_update_cart_quantity"}, {"name": "host", "value": "DESKTOP-BE07TO9"}, {"name": "thread", "value": "9988-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_update_cart_quantity"}]}